src/System.Private.CoreLib/src/System/Reflection/AssemblyName.cs

```diff
@@ -6,12 +6,14 @@
 using System.Runtime.Serialization;
 using System.Configuration.Assemblies;
 
-using Internal.Reflection.Augments;
-
 namespace System.Reflection
 {
     public sealed class AssemblyName : ICloneable, IDeserializationCallback, ISerializable
     {
+        private byte[] _publicKey;
```

Comment:
I think we can have all fields explicit here like it is in the CoreCLR version

Reply:
I have changed them with the guidelines

Reply:
Which guidelines?